src/DAMA/DoctrineTestBundle/Doctrine/DBAL/AbstractStaticConnection.php

```diff
@@ -50,6 +50,13 @@ public function rollBack(): bool
     }
 
     public function getWrappedConnection(): Connection
+    {
+        trigger_error('getWrappedConnection() is deprecated, call getNativeConnection() instead.', E_USER_DEPRECATED);
+
+        return $this->getNativeConnection();
+    }
+
+    public function getNativeConnection(): Connection
     {
         return $this->connection;
```

Comment:
this is not correct. This does not return the "native" connection.

We need to implement something similar to

https://github.com/doctrine/dbal/blob/3.3.x/src/Connection.php#L1542

So in case of using PDO this should return the actual `\PDO` connection instance

Reply:
You're right, I completely missed the point here.

Reply:
No worries :blush: This can be a bit complicated with all the different connection layers etc :yum: